mmgotool/commands/i18n.go

```diff
@@ -44,29 +51,61 @@ var CheckCmd = &cobra.Command{
 	Use:     "check",
 	Short:   "Check translations",
 	Long:    "Check translations existing in the source code and compare it to the i18n/en.json file",
-	Example: "  i18n list",
+	Example: "  i18n check",
 	RunE:    checkCmdF,
 }
 
+var CleanCmd = &cobra.Command{
+	Use:     "clean",
+	Short:   "Clean empty translations",
+	Long:    "Clean empty translations in translation files other than i18n/en.json base file",
+	Example: "  i18n clean",
+	RunE:    cleanCmdF,
+}
+
+var CleanAllCmd = &cobra.Command{
+	Use:     "clean-all",
+	Short:   "Clean all empty translations",
+	Long:    "Clean all empty translations in translation files other than i18n/en.json base file",
+	Example: "  i18n clean-all",
+	RunE:    cleanAllCmdF,
+}
+
 func init() {
 	ExtractCmd.Flags().Bool("skip-dynamic", false, "Whether to skip dynamically added translations")
 	ExtractCmd.Flags().String("portal-dir", "../customer-web-server", "Path to folder with the Mattermost Customer Portal source code")
 	ExtractCmd.Flags().String("enterprise-dir", "../enterprise", "Path to folder with the Mattermost enterprise source code")
 	ExtractCmd.Flags().String("mattermost-dir", "./", "Path to folder with the Mattermost source code")
+	ExtractCmd.Flags().Bool("contributor", false, "Allows contributors safely extract translations from source code without removing enterprise messages keys")
+
 	CheckCmd.Flags().Bool("skip-dynamic", false, "Whether to skip dynamically added translations")
 	CheckCmd.Flags().String("portal-dir", "../customer-web-server", "Path to folder with the Mattermost Customer Portal source code")
-	ExtractCmd.Flags().Bool("contributor", false, "Allows contributors safely extract translations from source code without removing enterprise messages keys")
 	CheckCmd.Flags().String("enterprise-dir", "../enterprise", "Path to folder with the Mattermost enterprise source code")
 	CheckCmd.Flags().String("mattermost-dir", "./", "Path to folder with the Mattermost source code")
 
+	CleanCmd.Flags().Bool("dry-run", false, "Run without applying changes")
+	CleanCmd.Flags().Bool("check", false, "Throw exit code on empty translation strings")
+	CleanCmd.Flags().String("file", "de.json", "Filename, e.g. de.json, to clean empty translation strings from")
```

Comment:
Same here, the default value should be empty or `en.json`, whatever you decide in the javascript side should be consistent with this.